src/codegen/generate-classes.ts

```diff
@@ -2191,6 +2191,8 @@ const GENERATED_CLASSES_HEADER = [
 
 #include "root.h"
 
+extern "C" bool Bun__isJSValueTransferable(JSC::JSGlobalObject* globalObject, JSC::JSValue encodedValue);
```

Comment:
also what if this existed in StructuredCloneSerialize::isTransferable instead? it's not called by zig

Reply:
sgtm